lib/modules/manager/pipenv/artifacts.ts

```diff
@@ -289,16 +139,11 @@ export async function updateArtifacts({
       await deleteLocalFile(lockFileName);
     }
     const cmd = 'pipenv lock';
-    const tagConstraint = await getPythonConstraint(
-      pipfileName,
-      newPipfileContent,
-      existingLockFileContent,
-      config,
-    );
-    const pipenvConstraint = getPipenvConstraint(
-      existingLockFileContent,
-      config,
-    );
+    const pipfileDir = getParentDir(ensureLocalPath(pipfileName));
+    const tagConstraint = await pipenvDetect.getPythonConstraint(pipfileDir);
```

Comment:
```suggestion
    const tagConstraint = config.constraints?.python ?? await pipenvDetect.getPythonConstraint(pipfileDir);
```
we should allow configure python version via constraints too

Reply:
Yep, I don't remember why I removed it, seems like by error